app/ts/components/App.tsx

```diff
@@ -38,6 +38,7 @@ export function App() {
 	const [isConnected, setIsConnected] = useState<IsConnected>(undefined)
 	const [useTabsInsteadOfPopup, setUseTabsInsteadOfPopup] = useState<boolean | undefined>(undefined)
 	const [currentTabId, setCurrentTabId] = useState<number | undefined>(undefined)
+	const [rpcEntries, setRpcEntries] = useState<RpcEntries>([])
```

Comment:
Still resisting `Signal`s?

Reply:
no, but I feel it requires bigger change to port over :/